SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -600,6 +603,35 @@ void USpatialNetDriver::QueryGSMToLoadMap()
 	GlobalStateManager->QueryGSM(QueryDelegate);
 }
 
+void USpatialNetDriver::OnActorSpawned(AActor* Actor)
+{
+	if (!Actor->GetIsReplicated())
+	{
+		return;
+	}
+
+	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton | SPATIALCLASS_NotSpatialType))
+	{
+		return;
+	}
+
+	if (Actor->GetLocalRole() != ROLE_Authority)
+	{
+		return;
+	}
+
+	if (USpatialStatics::IsActorGroupOwnerForActor(Actor))
+	{
+		return;
+	}
+
+	FString WorkerType = GetGameInstance()->GetSpatialWorkerType().ToString();
```

Comment:
Seems weird to me we don't grab this by ref?
```suggestion
	const FString WorkerType = GetGameInstance()->GetSpatialWorkerType().ToString();
```